x/bank/types/msgs.go

```diff
@@ -1,6 +1,8 @@
 package types
 
 import (
+	fmt "fmt"
```

Comment:
```suggestion
	"fmt"
```

Reply:
This import was removed as I switched `ValidateBasic` to return sdk errors instead of generic errors made by `fmt`.